README.md

```diff
@@ -37,10 +37,11 @@ Make sure you have the prerequisites:
 - Go 1.6
 - A running Kubernetes cluster
 - `kubectl` properly configured to talk to your cluster
-- Glide 0.10 or greater
+- [Glide](https://glide.sh/) 0.10 or greater with both git and mercurial installed.
```

Comment:
Git and Mercurial are not part of Glide's requirements -- they're part of Go's requirements.